hips/utils/wcs.py

```diff
@@ -124,6 +124,30 @@ def create(cls, skydir: SkyCoord, shape: tuple, coordsys: str = 'CEL',
 
         return cls(w, shape)
 
+    @classmethod
+    def create_simple(cls, skydir: SkyCoord, shape: tuple, fov_deg: float,
+                      coordsys: str = 'CEL', projection: str = 'AIT') -> 'WCSGeometry':
+        """Create WCS object programmatically using field of view (`WCSGeometry`).
+
+        Parameters
+        ----------
+        skydir : `~astropy.coordinates.SkyCoord`
+            Sky coordinate of the WCS reference point
+        shape : `tuple`
+            Shape of the image (Numpy axis order: y, x)
+        fov_deg : `float`
+            Field of view in degrees
+        coordsys : `str`
+            Coordinate system
+        projection : `str`
+            Projection of the WCS object
```

Comment:
Add an `Examples` section with an example (maybe the same one as you have for `create`?
I would also suggest that you use this more convenient method here:
https://hips.readthedocs.io/en/latest/getting_started.html#make-a-sky-image